### PR TITLE
[move] Change GAS to SUI in the framework

### PR DESCRIPTION
**Pull request by damirka**
As previously discussed, I'm adding this PR to hang for a bit until we got a green light on renaming.

### CODE REVIEW

**Comment by sblackshear** [comment]
LGTM, but double-checking that this is ok with you @gdanezis, since I know you had ideas about the timing of the rename + perhaps wanted both SUI and GAS coin types (I am not sure about the latter).

**Review by sblackshear — APPROVED** [review]
Double-checked with George on this, let's go forward!

File: sui_programmability/framework/sources/SUI.move [hunk 1]
```diff
@@ -2,26 +2,26 @@
 // SPDX-License-Identifier: Apache-2.0
 
 /// Coin<Gas> is the token used to pay for gas in Sui
```
Comment (awelc) [inline sui_programmability/framework/sources/SUI.move:4]: We should rename these as well.

Comment (damirka) [reply sui_programmability/framework/sources/SUI.move:4]: Whoopsie. Missed it it seems.

**Pull request merged**